api/registrations/serializers.py

```diff
@@ -95,6 +97,11 @@ def get_registered_meta(self, obj):
                 return meta_values
         return None
 
+    def get_embargo_end_date(self, obj):
+        if obj.embargo_end_date:
```

Comment:
Is there any reason you couldn't do:

```python
embargo_end_date = ser.DateTimeField(allow_null=True, read_only=True)
```

I imagine this would improve the auto-documentation of the serializer.